calibration/CTDMOG/CGINS-CTDMOG-13595__20170401.csv

```diff
@@ -22,4 +22,4 @@ serial,name,value,notes
 37-13595,CC_ptempa0,1.524068e+002,
 37-13595,CC_ptempa1,-6.926535e-002,
 37-13595,CC_ptempa2,5.813290e-007,
-37-13595,CC_wbotc,-2.1561e-007,
+37-13595,CC_wbotc,-2.156129e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT